plugin/src/main/java/io/jenkins/plugins/redis/RedisFingerprintStorage.java

```diff
@@ -37,15 +37,14 @@
 import java.io.StringWriter;
 import java.io.InputStream;
 import java.nio.charset.StandardCharsets;
+import java.util.ArrayList;
+import java.util.List;
 import java.util.logging.Logger;
 import java.util.logging.Level;
 
 import org.jenkinsci.main.modules.instance_identity.InstanceIdentity;
 
-import redis.clients.jedis.Jedis;
-import redis.clients.jedis.JedisPool;
-import redis.clients.jedis.JedisPoolConfig;
-import redis.clients.jedis.Transaction;
+import redis.clients.jedis.*;
```

Comment:
Check the style guide for Jenkins projects; it might want us to expand the imports out.

Reply:
Is this where you set the limit? https://github.com/jenkinsci/redis-fingerprint-storage-plugin/pull/23/files#diff-5a1ae3679c7223b516c0e27d1e4ca65fR168